tidy3d/components/medium.py

```diff
@@ -539,10 +563,48 @@ def eps_diagonal_on_grid(
         ]
         return tuple(eps_list)
 
+    def eps_comp_on_grid(
```

Comment:
Just curious, where is this method used? in the backend?

Reply:
so, it's used in `Structure.eps_comp(...)` which is used in `Simulation.epsilon_on_grid(...)` which is used in `Simulation.epsilon(...)` and in mode solver